internal.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+# This script is used by crc developer or internal CI to build the KAO/KCMO patch images to
+# brew and then mirror built container images to quay.io/crcont. Pull secret should contain
+# the auth for `quay.io/crcont` before providing to this script.
+#    - Since this script uses rhpkg and kinit commands, it is tested on linux (not mac/windows)
+#    - As of now this script works with 4.11 releases because only `rhaos-4.11-rhel-8` branch
+#      created in dist-git and tested.
+# Usage:
+# If you want to build latest candidate stream for 4.11
+#   - ./internal.sh
+# If you want to build specific version of 4.11.3
+#   - OPENSHIFT_VERSION=4.11.3 ./internal.sh
+
+set -exuo pipefail
+
+export LC_ALL=C.UTF-8
+export LANG=C.UTF-8
+
+# kill all the child processes for this script when it exits
+trap 'jobs=($(jobs -p)); [ -n "${jobs-}" ] && ((${#jobs})) && kill "${jobs[@]}" || true' EXIT
+
+rm -fr crc-cluster-kube-apiserver-operator
+rm -fr crc-cluster-kube-controller-manager-operator
+
+function run_preflight_checks() {
```

Comment:
You can probably drop the function, or give it a more specifc name? (`check_pull_secret`)

Reply:
Agree, used `check_pull_secret`